lib/mongo.ex

```diff
@@ -1129,7 +1129,29 @@ defmodule Mongo do
 
   e.g. long-hand `query` becomes short-hand `q`, snake case `array_filters`
   becomes `arrayFilters`
+
+  Example:
+
+      Mongo.update(MongoPool,
+        "test_collection",
+        query: %{foo => 4},
+        update: %{"$set": %{"modified_field": "new_value"}},
+        multi: true)
+
+        Mongo.update(MongoPool,
+          "test_collection",
+          query: %{foo: 4},
+          update: %{foo: 5, new_field: "new_value"}},
+          upsert: true)
+
+        Mongo.update(MongoPool, "test_collection", [
+          [q: %{foo: 24}, update: %{flag: "old"}],
+          [q: %{foo: 99}, update: %{luftballons: "yes"}, upsert: true]
+        ])
```

Comment:
Do these examples suffice? I think this shows the most common ways of supplying updates, WDYT?